index_bg.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: index
+lang: bg
+subtitle: Мениджър на пакети за OS X
+font-family: helvetica
+
+pagecontent:
+  question: Какво представлява Homebrew?
+  what: Homebrew инсталира <a href="https://github.com/Homebrew/homebrew/tree/master/Library/Formula" title="Списък с пакети за Homebrew">всичко необходимо</a>, което Apple не може.
+  how: Homebrew инсталира пакетите в неговата директория и след това създава линкове към техните файлове в <code>/usr/local</code>.
+  prefix: Можете да инсталирате Homebrew където пожелаете. Той няма да инсталира файлове извън зададената директория.
+  createpackages: Създаването на Homebrew пакети е лесно и просто.
+  hack: Под капака е rubi и git, така че с лекота можете да правите промени.
+  formula: Homebrew формулите са прости Ruby скриптове:
+  editor: opens in $EDITOR!
```

Comment:
Nope, it was this way in some other languages like Russian, so I left it this way.